Bot google spreadsheet (#40)
callbellchat/callbell-public-docs: i18n/pt/docusaurus-plugin-content-docs/current/bot/editor/actions/logic/google_spreadsheet.md

# Planilha do Google

Essa ação permite que seu bot interaja com um documento do Planilhas Google inserindo, atualizando e selecionando linhas dinamicamente durante as interações do usuário. Ela pode ser usada para armazenar, recuperar e modificar dados coletados de conversas de bots.

# Casos de uso:

- Armazenamento das respostas do usuário diretamente em um arquivo do Planilhas Google para processamento posterior.
- Atualização de entradas específicas no Planilhas Google com base na entrada do usuário.
- Recuperação de dados de uma planilha do Google para uso em uma conversa ou como condições no fluxo do bot.

# Como usar:

## 1. Adicionando uma ação do Google Sheets

Para adicionar uma ação do Google Sheets no editor de fluxo da Callbell, siga estas etapas:

1. Abra o Flow Builder e selecione o ponto da conversa em que você deseja integrar o Google Sheets.
2. Adicione uma nova ação clicando no botão Add Action (Adicionar ação) e selecionando Google Sheets na lista de ações disponíveis, dentro da seção "Logic" (Lógica).

## 2. Faça a autenticação em sua conta do Google

Clique no botão "Google connect" e siga as instruções.

:::tip

> Certifique-se de selecionar todos os direitos. O bot da Callbell não funcionará corretamente se o direito "Google Drive File" ou "Google Spreadsheet Sheet" estiver ausente.

Reviewer: ```suggestion
Certifique-se de selecionar todas as permissões. O bot da Callbell não funcionará corretamente se a permissão "Google Drive File" ou "Google Spreadsheet Sheet" estiver ausente.
```
